protocols/purple/purple.c

```diff
@@ -364,6 +368,14 @@ static void purple_sync_settings(account_t *acc, PurpleAccount *pa)
 		const char *name = "line-auth-token";
 		purple_account_set_string(pa, name, set_getstr(&acc->set, name));
 	}
+
+        PurpleProxyInfo *proxyinfo = purple_account_get_proxy_info(pa);
+        if (!proxyinfo) {
+          proxyinfo = purple_proxy_info_new();
+        }
+        assert (proxyinfo);
```

Comment:
We don't normally use assert (with break), usually just `g_return_if_fail` or similar. What are you asserting here anyway? If it's out of memory then just don't.

Reply:
Thanks, I figured that wasn't the right thing to do, but wasn't sure what to do instead.

Reply:
Squash that with the previous commit?

Reply:
I normally just do that on merge

Reply:
What about merging this?